### PR TITLE
Add instructions for local prettier linting

### PR DESCRIPTION
**Pull request by npentrel**
# Description

Please include a summary of the change and what is does. Please also include relevant motivation and context.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,6 +78,25 @@ done
 fi
 ```
 
+### Lint JS and Markdown files with Prettier on save
+
+1. Install the [Prettier VS Code Extension](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode).
+2. Run `npm install` in the docs folder you have the docs checked out in.
+3. Inside VS code, open `settings.json` and ensure the following settings are in the file:
```
Comment (sguequierre) [inline README.md:85]: how do you get to `settings.json`? On my macbook I can only open up a GUI settings, tried searching how to do it to no avail

**Review by sguequierre — APPROVED** [review]
LGTM one comment for own confusion

File: README.md (hunk 1 above)
Comment (npentrel) [inline README.md:85]:
```suggestion
3. Inside VS code, open `settings.json` by pressing `CMD+SHIFT+P` and typing in settings and ensure the following settings are in the file:
```

Comment (npentrel) [reply README.md]: updated!

**Pull request merged**